client/src/components/general/stats.tsx

```diff
@@ -5,11 +5,25 @@ import { Loader } from "lucide-react";
 import { useMemo } from "react";
 import { z } from "zod";
 
-import { GetStatisticsDocument } from "~/generated/graphql/graphql";
+import { GetStatisticsDocument, GetCachedStatisticsDocument } from "~/generated/graphql/graphql";
+
+import { Button } from "../ui/button";
 
 import { Chart } from "./chart";
 
 
+export const cachedStatistics = z.object({
+  number_of_project: z.number(),
```

Comment:
Personal preference and generally a JavaScript standard: can we use camelCase for these variables?